examples/ota-requestor-app/linux/main.cpp

```diff
@@ -85,6 +85,9 @@ static void InitOTARequestor(void)
     gRequestorCore.Init(&(chip::Server::GetInstance()), &gRequestorUser, &gDownloader);
     gRequestorUser.Init(&gRequestorCore, &gImageProcessor);
 
+    // Set the default providers timeout to one minute
+    gRequestorUser.SetDefaultProvidersTimeoutSec(60);
```

Comment:
We should make this configurable. If not in this PR, at least a todo.

Reply:
Actually, you could just hijack the -q option above. The value is already read/stored. You just need to pass it in here and update the help text and readme.

Reply:
Done